examples/with-mobx/pages/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Provider } from 'mobx-react'
+import { initStore } from '../store'
+import Clock from '../components/Clock'
+
+export default class Counter extends React.Component {
+  static getInitialProps ({ req }) {
+    const isServer = !!req
+    const store = initStore(isServer)
```

Comment:
Try add a comment on why we do this? ( I mean in the isServer part)
And why we need to init our store here.